src/Symfony/Component/ErrorRenderer/composer.json

```diff
@@ -7,8 +7,8 @@
     "license": "MIT",
     "authors": [
         {
-            "name": "Fabien Potencier",
-            "email": "fabien@symfony.com"
+            "name": "Yonel Ceruto",
+            "email": "yonelceruto@gmail.com"
         },
```

Comment:
I think you should keep me and add you as another author?

Reply:
Fixed